docs/Configuration.md

```diff
@@ -1069,6 +1069,12 @@ Default: `failure-change`
 
 Specifies notification mode. Requires `notify: true`.
 
+### `openHandlesTimeout` \[number]
```

Comment:
Move it below Modes please. These belong to `notifyMode `.

Reply:
Oops - thanks